components/style/error_reporting.rs

```diff
@@ -27,10 +28,14 @@ pub trait ParseErrorReporter {
 /// TODO(emilio): The name of this reporter is a lie, and should be renamed!
 pub struct StdoutErrorReporter;
 impl ParseErrorReporter for StdoutErrorReporter {
-    fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str) {
+    fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str, servo_url : Option<&ServoUrl>) {
          if log_enabled!(log::LogLevel::Info) {
              let location = input.source_location(position);
-             info!("{}:{} {}", location.line, location.column, message)
+             match  servo_url {
```

Comment:
[`map`](https://doc.rust-lang.org/std/option/enum.Option.html#method.map) is called only when there's `Some(T)` in `Option<T>`, and `unwrap_or("")` makes sure that if there's `None`, then we'll get an empty string. So, we're good :)